APMrover2/GCS_Mavlink.cpp

```diff
@@ -725,7 +741,7 @@ void GCS_MAVLINK_Rover::handleMessage(mavlink_message_t* msg)
         }
         mavlink_statustext_t packet;
         mavlink_msg_statustext_decode(msg, &packet);
-        char text[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN+1+4] = { 'G','C','S',':'};
+        char text[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN+1+4] = { 'G', 'C', 'S', ':'};
```

Comment:
I think that it is easier to distinguish clearly from the variable if you insert a space  in +1+4.